services/dqa-secondary-reports/README.md

```diff
@@ -0,0 +1,35 @@
+# Service for DQA secondary report information
+
+Service that exposes a endpoint for fetching PEDSnet DQA secondary report information for different data model versions, e.g.
+
+```
+/pedsnet/2.0.0
+/pedsnet/1.0.0
+```
+
+## Installation
+
+To install the prerequisites, run
+```
+pip3 install -r requirements.txt 
+```
+
+## Run
+
+Usage:
+```
+python3 main.py [--token=<token>] [--host=<host>] [--port=<port>] [--debug] [--interval=<interval>]
+```
+
+To get the help message, run
+```
+python3 main.py -h
+```
```

Comment:
It would be better to combine the two lines to just:

```
To get usage information, run:

python3 main.py -h
```

since typing out the usage line above could become inconsistent over time.